index.js

```diff
@@ -6,52 +6,20 @@ import './shim.js';
 
 import React from 'react';
 import { AppRegistry, StatusBar } from 'react-native';
-import SplashScreen from 'react-native-splash-screen';
 
 import App from './App';
-import WalletMigrate from './walletMigrate';
 
 if (!Error.captureStackTrace) {
   // captureStackTrace is only available when debugging
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
   Error.captureStackTrace = () => {};
 }
 
-class BlueAppComponent extends React.Component {
-  state = {
-    isMigratingData: true,
-  };
-
-  componentDidMount() {
-    const walletMigrate = new WalletMigrate(this.setIsMigratingData);
-
```

Comment:
don't we need it?

Reply:
seems to be useless, I think it was a migration that BlueWallet needed. Do you know for what we need it?